src/Pretzel.Logic/Templating/Context/PathExtensions.cs

```diff
@@ -5,7 +5,7 @@ namespace Pretzel.Logic.Templating.Context
 {
     public static class PathExtensions
     {
-        private static readonly string[] MarkdownFiles = new[] { ".md", ".mdown", ".markdown" };
+        private static readonly string[] MarkdownFiles = new[] { ".md", ".mkd", ".mkdn", ".mdown", ".markdown" };
```

Comment:
:thumbsup: to the change, but this seems like a [very big yak](http://superuser.com/a/285878) to shave :grinning: